### PR TITLE
[ch33583] Improve slow loaders-phase II

### PR DESCRIPTION
**Pull request by hbarisik**
The updated loaders make minimum number of querries to source db (etools db). Particularly, extra queries  per source record have been targeted to eliminate.

Intervention related loaders do not perform any extra query per source record to source db (etools db) anymore.  Now, any remaining slowness are due target db (datamart db ) queries and updates.

Bulk update is attempted but left as an additional level of improvements to keep this update simpler.

### CODE REVIEW

File: src/etools_datamart/apps/mart/data/models/partners_interventionbudget.py [hunk 1]
```diff
@@ -44,23 +173,6 @@ def process_country(self):
                 op = self.process_record(filters, values)
                 self.increment_counter(op)
 
-    def get_fr_numbers(self, record: PartnersIntervention, values: dict, **kwargs):
```
Comment (emaciupe) [inline src/etools_datamart/apps/mart/data/models/partners_interventionbudget.py:47]: @hbarisik why was this removed?

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -85,31 +83,38 @@ services:
...
-    shm_size: '1gb'
-    container_name: datamart_prp
-    environment:
-      POSTGRES_HOST_AUTH_METHOD: trust
-      #POSTGRES_USER: postgres
-      #POSTGRES_PASSWORD:
-      POSTGRES_DB: prp
-    volumes:
-      - "$PWD/build/prp:/var/lib/postgresql/data"
+  #NOTE: prp db server can be launched externally
```
Comment (emaciupe) [inline docker-compose-dev.yml:104]: @hbarisik please remove local setup changes

**Review by emaciupe — CHANGES_REQUESTED** [review]
@hbarisik I wouldn't remove the get_fr_numbers method unless it was requested - as perhaps it is used.

File: src/etools_datamart/apps/mart/data/models/partners_interventionbudget.py (hunk 1 above)
Comment (hbarisik) [reply src/etools_datamart/apps/mart/data/models/partners_interventionbudget.py:47]: Restored with updated implementations. Thank you :-)

**Review by emaciupe — APPROVED** [review]

**Pull request merged**